frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,211 @@
-  font-weight: bolder;
-  `}
-`;
-
-const TrendingIcon = styled(TrendingUpIcon)`
-  fontsize: 20px;
-  marginleft: 10px;
-`;
-
-const NavigationLink = styled(RouterLink)`
-  ${({ theme }) => `
-  color: ${theme.palette.text.secondary};
-  `}
-`;
-
-interface HeaderProps {
-  onNavigate: () => void;
-}
-
-const Header: React.FC<HeaderProps> = ({ onNavigate }) => {
-  return (
-    <DrawerHeader container flex-wrap="wrap">
-      <Grid item>
-        <NavigationLink to="/" onClick={onNavigate} data-qa="logo">
-          <Logo />
-        </NavigationLink>
-      </Grid>
-      <Grid item>
-        <Grid container justify="center" direction="column">
-          <Typography
-            style={{ lineHeight: "1.3", margin: "-5px 0px 10px 10px" }}
-            component="span"
-            color="primary"
-            data-qa="title"
-          >
-            <NavigationLink to="/" onClick={onNavigate}>
-              <GroupHeading>clutch</GroupHeading>
-            </NavigationLink>
-            <Typography variant="caption">Welcome {userId()}!</Typography>
-          </Typography>
-        </Grid>
-      </Grid>
-    </DrawerHeader>
-  );
-};
-
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.accent.main};
-  `}
-`;
+// sidebar
+const drawerWidth = "100px";
+
+const DrawerPanel = styled(MuiDrawer)({
+  flexShrink: 0,
+  width: `${drawerWidth}`,
+  "div[class*='MuiDrawer-paper']": {
+    width: `${drawerWidth}`,
+    top: "64px",
+    backgroundColor: "#FFFFFF",
+    boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
+  },
+});
+
+// sidebar groupings
+const GroupList = styled(List)({
+  padding: "0px",
+});
+
+const GroupListItem = styled(ListItem)({
+  flexDirection: "column",
+  height: "82px",
+  padding: "16px 8px 16px 8px",
+  "&:hover": {
+    backgroundColor: "#F5F6FD",
+  },
+  "&:active": {
+    backgroundColor: "#D7DAF6",
+  },
+  // avatar and label
+  "&:hover, &:active, &.Mui-selected": {
+    ".MuiAvatar-root": {
+      backgroundColor: "#3548D4",
+    },
+    ".MuiTypography-root": {
+      color: "#3548D4",
+    },
+  },
+  "&.Mui-selected": {
+    backgroundColor: "#FFFFFF",
+    "&:hover": {
+      backgroundColor: "#F5F6FD",
+    },
+    "&:active": {
+      backgroundColor: "#D7DAF6",
+    },
+  },
+});
+
+const GroupHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+  flexGrow: 1,
+  paddingTop: "11px",
+});
+
+const Avatar = styled(MuiAvatar)({
+  background: "rgba(13, 16, 48, 0.6)",
+  height: "24px",
+  width: "24px",
+  color: "#FFFFFF",
+  fontSize: "14px",
+});
+
+// sidebar submenu
+const Popper = styled(MuiPopper)({
+  zIndex: 1200,
+  paddingTop: "16px",
+});
+
+const Paper = styled(MuiPaper)({
+  width: "230px",
+  border: "1px solid #E7E7EA",
+  boxShadow: "0px 10px 24px rgba(35, 48, 143, 0.3)",
+
+  // sidebar submenu groupings
+  ".MuiListItem-root[data-qa='workflowGroupItem']": {
```

Comment:
if you style a component that uses the [`component`](https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/AppLayout/drawer.tsx#L178) prop, you get a TS error. there's a [material ui issue](https://github.com/mui-org/material-ui/issues/22452) on this. workaround was to use the [`data-qa='workflowGroupItem` selector](https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/AppLayout/drawer.tsx#L182)